assembly/README.md

```diff
@@ -28,7 +28,7 @@ Pushing to a specific docker registry under a specific account:
 ### Run
 
     docker run -td --name kapua-sql -p 8181:8181 -p 3306:3306 kapua/kapua-sql
-    docker run -td --name kapua-elasticsearch -p 9200:9200 -p 9300:9300 elasticsearch:2.4 -Dcluster.name=kapua-datastore -Ddiscovery.zen.minimum_master_nodes=1
+    docker run -td --name kapua-elasticsearch -p 9200:9200 -p 9300:9300 elasticsearch:5.3.0 -Ecluster.name=kapua-datastore -Ediscovery.zen.minimum_master_nodes=1 -Enetwork.host=0.0.0.0 
```

Comment:
@lorthirk that works for me as well, so go ahead with the fix. Out of curiosity, which version of docker are you using?